.prettierrc.json

```diff
@@ -1 +1,6 @@
-{}
+{
+  "trailingComma": "all",
+  "tabWidth": 2,
+  "semi": false,
+  "singleQuote": true
```

Comment:
We are not allowing semicolons...?
Just confirming

Reply:
what would you suggest?

Reply:
I guess let it be there
Since this repo is beginner friendly, let them have a habit of putting semicolons.
If we don't allow semicolons, they might get confused.

My opinion 👆🏻

Reply:
Sure!